controller/nodegroup_test.go

```diff
@@ -0,0 +1,132 @@
+package controller
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/eks"
+	v1 "github.com/rancher/eks-operator/pkg/apis/eks.cattle.io/v1"
+	"sort"
+	"testing"
+
+
+	"github.com/stretchr/testify/assert"
```

Comment:
nit (personal opinion): I don't really like testify, especially when only used for asserts.

Reply:
We use it all over rancher. Do you just think it's not worth bringing in for assert only? Why don't you like testify?

Reply:
I know it is used in a lot of Go packages. I haven't found that it provides me a lot of value, especially assert. Error handling in Go is very verbose, and using assert feels like it's from a different language.